apisyouwonthate.yml

```diff
@@ -72,12 +72,42 @@ rules:
           maxItems: 1
 
   # Author: Andrzej (https://github.com/jerzyn)
-  request-GET-no-body:
```

Comment:
Did you decide to ditch this one intentionally?

Reply:
eee, honest mistake! don't want to get rid of this! :o